app/controllers/hiring_staff/sign_in/dfe/sessions_controller.rb

```diff
@@ -18,8 +19,13 @@ def create
   end
 
   def destroy
+    if session[:signing_out_for_inactivity]
+      flash_message = { notice: I18n.t('messages.access.signed_out_for_inactivity') }
+    else
+      flash_message = { success: I18n.t('messages.access.signed_out') }
+    end
     session.destroy
-    redirect_to root_path, notice: I18n.t('messages.access.signed_out')
+    redirect_to new_identifications_path, flash_message
```

Comment:
I think perhaps this should be `return redirect_to`, in order to make the action `destroy` closed and thus avoid the possibility of too many redirects.

Reply:
No need to return on the last line of the method